resources/views/macros/form.twig

```diff
@@ -77,3 +77,8 @@
 {% macro submit(label, opt) %}
     {{ _self.button(label|default(__('form.submit')), opt|default({})|merge({'type': 'submit'})) }}
 {%- endmacro %}
+
+{% macro info(text) %}
+{% import 'macros/base.twig' as m %}
```

Comment:
Hmm, imho it should be imported at the top? And i would also move it to the base macros as its not that form specific imho

Reply:
That didn't work. Any other ideas?

Reply:
0o

Thats interesting (and looks more like a bug than a feature..)

But would be irrelevant if moved to the `base.twig` file

Reply:
According to https://stackoverflow.com/questions/9548601/calling-a-macro-inside-another-macro-in-twig, the import was actually the correct way. It's even needed when moved to `base.twig`.

Reply:
When referencing macros from the same file the `_self` keyword can be used ;)